src/ci/citool/Cargo.toml

```diff
@@ -19,3 +21,8 @@ insta = "1"
 # If this is omitted, cargo will look for a workspace elsewhere.
 # We want to avoid this, since citool is independent of the other crates.
 [workspace]
+
+# Make compilation as fast as possible on CI
+[profile.dev]
+debug = 0
+incremental = false
```

Comment:
should we set `CARGO_INCREMENTAL=0` in CI so that locally you still get incremental compilation? 🤔

Reply:
Non-incremental local debug rebuild of this crate is 0.6s on my PC, incremental is 0.4s. I don't think that incremental is really needed here. But I can switch to the env. var. if you would prefer it.

Reply:
I prefer the env var because there could be people with less powerful hardware, and this program could become more complex in the future